robohive/envs/multi_task/common/kitchen/franka_kitchen.xml

```diff
@@ -54,6 +54,7 @@
 
 
         <!-- Robot -->
+        <site name='ee_target' type='box' size='.03 .07 .04' pos='-0.4 0.4 2.1' group='1' rgba='0 1 .4 0' euler="0 3.14 3.14"/>
```

Comment:
There is already a site called target that you can reuse 
`<site name='target' pos='0 0 0' size='0.1' rgba='0 2 0 .2'/>`
for teleOp targets.

Reply:
Currently, `target` is set at the origin. I thought I'd add another `ee_target` to keep it consistent with the other environment. What do you suggest?

Reply:
It's set at a neutral position during initialization. Feel free to move it around as needed.